apps/condo/domains/billing/access/BillingCategory.js

```diff
@@ -12,19 +12,10 @@ async function canReadBillingCategories ({ authentication: { item: user } }) {
     return {}
 }
 
-async function canManageBillingCategories ({ authentication: { item: user } }) {
```

Comment:
? why did you just delete the Manage check ?

Reply:
Nobody can create or update billing categories now

Reply:
Returned the manage check

Reply:
Returned the manage check

Reply:
returned